crates/templates/src/manager.rs

```diff
@@ -181,6 +191,15 @@ impl TemplateManager {
         };
         let id = template.id();
 
+        if !install_template_list.is_empty() && !install_template_list.contains(&id.to_owned()) {
```

Comment:
This makes the empty vector a magic value that means "all" rather than "none."  Consider an enum with appropriate methods.

(One of those "the CLI / serialisation / etc. representation is not necessarily the correct _parsed_ representation" situations.)

Reply:
Good point. Will make the changes required.